doc/source/examples/multiphysics/sloshing-in-rectangular-tank/sloshing-in-rectangular-tank.rst

```diff
@@ -174,12 +179,13 @@ to run the simulation using eight CPU cores. Feel free to use more.
     run in parallel using mpirun. This simulation takes
     :math:`\sim \, 8` minutes (:math:`Re=2`) to :math:`6` hours (:math:`Re=2000`) on :math:`8` processes.
 
+****
 
 -------
 Results
 -------
 
-We compare the relative height of the free surface at :math:`x=0` with an analytical solution proposed by Wu *et al.* [#wu2001]_ For the Reynolds number of :math:`2`, :math:`20` and :math:`200`, data were directly extracted from Carrica *et al.* [#carrica2007]_, whereas for the Reynolds of :math:`2000`, the simplified analytical expression of Wu *et al.* [#wu2001]_ is used. The results for Reynolds number of :math:`2`, :math:`20`, :math:`200` and :math:`2000` can be post-processed by invoking the following command from the folder of the Reynolds number of interest (:math:`Re=20` in the example below):
+We compare the relative height of the free surface at :math:`x=0` with an analytical solution proposed by Wu *et al.* [#wu2001]_ For the Reynolds number of :math:`2`, :math:`20` and :math:`200`, data were directly extracted from Carrica *et al.* [#carrica2007]_, whereas for the Reynolds of :math:`2000`, the simplified analytical expression of Wu *et al.* [#wu2001]_ is used. The results for Reynolds number of :math:`2`, :math:`20`, :math:`200` and :math:`2000` can be postprocessed by invoking the following command from the folder of the Reynolds number of interest (:math:`Re=20` in the example below):
```

Comment:
Both spelling are acceptable. I am more a fan of the hyphen version, but anyway
https://english.stackexchange.com/questions/452969/pre-processing-post-processing-hyphen-presence-in-research-publications-writ